connections/oikotie/mapper.py

```diff
@@ -0,0 +1,358 @@
+from datetime import date, timedelta
```

Comment:
There are a few things, like eur/kk, m2 and eur that are used in couple of places, maybe could have them defined in one place, but not sure if worth it?

Reply:
Good point! I'll define those in one place because they are constant/vakio